scripts/generator/graphql_generator/csharp/SDK/iOS/ApplePayEventResponse.cs.erb

```diff
@@ -15,6 +15,14 @@ namespace <%= namespace %>.SDK.iOS {
             SummaryItems = summaryItems ?? new List<SummaryItem>();
             ShippingMethods = shippingMethods ?? new List<ShippingMethod>();
         }
+
+        public string ToJsonString() {
+            var dict = new Dictionary<string, object>();
+            dict["SummaryItems"] = SummaryItems;
+            dict["AuthorizationStatus"] = AuthorizationStatus;
+            dict["ShippingMethods"] = ShippingMethods;
```

Comment:
Same thing with `ShippingMethods` it needs to be a list of `List<Dictionary<string, object>>`.